code/src/layouts/dashboard-layout/components/dashboard-admin-tab-layout/DashboardAdminLayout.constants.tsx

```diff
@@ -1,9 +1,15 @@
 import PeopleIcon from "@mui/icons-material/People";
+import ReceiptLongIcon from "@mui/icons-material/ReceiptLong";
 
 import { DASHBOARD_TAB_NAMES } from "@/layouts/dashboard-layout/DashboardLayout.constants";
 import { DashboardTab } from "@/layouts/dashboard-layout/DashboardLayout.types";
 
 export const dashboardAdminTabs: DashboardTab[] = [
+  {
+    labelTranslationKey: "dashboardTabs.orders.label",
+    name: DASHBOARD_TAB_NAMES.ORDERS,
+    icon: <ReceiptLongIcon />
+  },
   {
     labelTranslationKey: "dashboardTabs.users.label",
     name: DASHBOARD_TAB_NAMES.USERS,
```

Comment:
I think users tab should be the first in the list. What do you think?

Reply:
I agree with you. Fixed!

Reply:
![image](https://github.com/user-attachments/assets/8cbe9501-9873-46af-95ae-231ff32ca1ec)